common-lib/common/signal/test_counter.libsonnet

```diff
@@ -38,7 +38,7 @@ local m1 = signal.init(
           expect: 'API server requests',
         },
         testUnit: {
-          actual: m1.asTimeSeries().fieldConfig.defaults.unit,
+          actual: m1.asTimeSeries().fieldConfig.overrides[0].properties[1].value,
```

Comment:
minor nit here as `overrides[0].properties[1].value` is not quite as readable as `defaults.unit`.

Would be good to have a comment for future maintainers explaining just what field is being looked at and why it's done in this manner

Reply:
There is a global place to to put units into any panel (fieldConfig.defaults.unit). But it will be used by multiple signals you can attach to a panel.
So in commonlib I decided to move units into overrides in order to make sure that units setting only applies to the specific signal. So that is why so long path in unit tests.